mono/mini/mini-runtime.c

```diff
@@ -3470,6 +3477,27 @@ MONO_SIG_HANDLER_FUNC (, mono_sigsegv_signal_handler)
 #endif
 }
```

Comment:
can we get rid of `MONO_SIG_HANDLER_DEBUG` and `MONO_SIG_HANDLER_FUNC_DEBUG`, and make `mono_sigsegv_signal_handler ()` always call `mono_sigsegv_signal_handler_debug ()`?

move all the logic if it works on a certain platform into `mono_sigsegv_signal_handler ()`. if it isn't supported/working, just pass `NULL`.

Reply:
I did that deliberately so that NULL would not appear as the faulting address incorrectly.
I want no faulting address to appear if we don't have it.

Reply:
yeah (I edited my comment): A magic value would be a good fit then.

Reply:
That still makes me a little nervous.
Have a *slightly* smaller diff locally but still haven't unconditionally passed the extra parameter.